draft-ietf-scitt-architecture.md

```diff
@@ -316,7 +316,7 @@ Considering CT in terms of SCITT:
 - CAs (Issuers) sign the ASN.1 DER encoded tbsCertificate structure to produce an X.509 certificate (Signed Statements)
 - CAs submit the certificates to one or more CT logs (Transparency Services)
 - CT logs produce Signed Certificate Timestamps (Transparent Statements)
-- Signed Certificate Timestamps are checked by Relying Parties
+- Signed Certificate Timestamps, Signed Tree Heads, and their respective consistency proofs are checked by Relying Parties
```

Comment:
The arch doc doesn't discuss Signed Tree Heads. Proposing:
```suggestion
- Signed Certificate Timestamps and their respective consistency proofs are checked by Relying Parties
```

Reply:
Steve, Does this mean a Transparency Service provider MUST check Signed Certificate Timestamps and their respective consistency proofs?

Reply:
Good catch, @rjb4standards:

> consistency proofs are checked by Relying Parties

questions if "are" is a MUST, SHOULD, COULD, or just a passive reference.

As this is the ["Definition of Transparency"](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#section-3) section, I would read this as an example and IMO, I'm ok with it not being more restrictive. For that matter, x.509 is not required, so I wouldn't take this further.

Reply:
Leaving Signed Tree Heads
We may want to add more information related to CT in the Definition of Transparency section. ...in another PR